### PR TITLE
add support for transferring NFT

### PR DESCRIPTION
**Pull request by robertu7**

### CODE REVIEW

File: src/components/Dialogs/TransferDialog/CompleteContent/index.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  const txHash = txReceipt.transactionHash
+  const { url, maskedHash } = toEtherscanUrl(txHash)
+
+  return (
+    <>
+      <Dialog.Content>
+        <p className={styles.content}>
+          {locale === Lang.en
+            ? "Now your friends will receive gifts in their list, and they're available to read the contents of Logbook."
```
Comment (robertu7) [inline src/components/Dialogs/TransferDialog/CompleteContent/index.tsx:32]: @devformatters copy needs your confirmation.

File: src/components/Dialogs/TransferDialog/CompleteContent/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            : "Now your friends will receive gifts in their list, and they're available to read the contents of Logbook."}
```
Comment (tx0c) [inline src/components/Dialogs/TransferDialog/CompleteContent/index.tsx:33]: Looks good;  the translations can always be addressed in another PR revision

**Review by tx0c — APPROVED** [review]
